src/utils/network/session-core.js

```diff
@@ -293,3 +305,72 @@ export const getBlobFromSolid = async (session, fileUrl) => {
   const fileBlob = await getFile(fileUrl, { fetch: session.fetch });
   return URL.createObjectURL(fileBlob);
 };
+
+/**
+ * A function fetches the user's profile information from their webId's profile
+ * card
+ *
+ * @function fetchProfileInfo
+ * @param {Session} session - Solid's Session Object {@link Session}
+ * @returns {Promise<object>} Object - The object containing the information related
+ * to the person on their profile card, the profile dataset, and the profile Thing
+ */
+export const fetchProfileInfo = async (session) => {
```

Comment:
Can we move these profile functions into separate files? Like how we have for documents and user list. Dumping all the business logic into a single monolithic file like `session-core` is pretty bad practice.

Reply:
Moving `fetchProfileInfo`, `updateProfileInfo`, and `updateActivity` functions into `profile-helper.js` inside `utils/network`